cmd/copy_sync_windows_test.go

```diff
@@ -0,0 +1,382 @@
+//go:build windows
+
+package cmd
```

Comment:
This feels the same way. The test itself isn't reliant upon anything that doesn't compile into the executable, so maybe items in the test matrix could be flagged with a target GOOS?